controls/C-0262-anonymousaccessisenabled.json

```diff
@@ -0,0 +1,15 @@
+{
+    "controlID": "C-0262",
```

Comment:
This should be added to the security framework!

Reply:
In addition to AllControls, right? We want it to be available as a compliance check

Reply:
Yes, in addtion